README.md

```diff
@@ -55,3 +55,30 @@ to the cloud.gov IdP provider. This route is used to set the user's origin to
 the user, `cg-uaa-extras` will redirect to the `IDP_PROVIDER_URL` to complete
 the user's authentication and TOTP token creation. This is why the URL from the
 screenshot above is necessary for the `IDP_PROVIDER_URL`.
+
+### Development
+
+This project uses a `setup.py` to install dependencies. Run the following
+command to get started with development.
+
+```shell
+python3 ./setup.py install
+```
+
+To get a local server up, run the following command. Make sure you
+properly setup the environment variables mentioned above in the
+documentation.
+
+```shell
+python3 ./run.py
+```
```

Comment:
Good call. I'm stretching my "sealegs" a bit on this project as my first dive into Python. Hadn't quite dug into `debug.py` since I didn't see a `requirements.txt` file and wasn't really sure how to install the dependancies. Once I discovered `python3 setup.py install`, I sort of just forgot what I was doing. 😆 Good catch! I'll update it. Thanks.